packages/ia-topnav/src/styles/media-button.js

```diff
@@ -43,6 +41,10 @@ export default css`
     -ms-flex-pack: center;
     justify-content: center;
   }
+  .menu-item > .icon > svg {
+    height: 4rem;  
+    width: 4rem;  
```

Comment:
4rem is prominent i've seen this in multiple areas, can we make this a css var with 4rem default?